packages/lib-classifier/src/plugins/tasks/DrawingTask/components/tools/DrawingToolRoot.js

```diff
@@ -0,0 +1,56 @@
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+
+const STROKE_WIDTH = 1.5
+const SELECTED_STROKE_WIDTH = 2.5
+
+function DrawingToolRoot ({ children, tool }) {
```

Comment:
Non-blocking suggestion: Maybe this can be written as a HOC instead and decorate the marks?

Reply:
See #1308.